src/ansys/dpf/core/operators/averaging/force_summation_psd.py

```diff
@@ -186,12 +183,6 @@ def _spec() -> Specification:
                 ),
             },
             map_output_pin_spec={
```

Comment:
@oparreno @ecottanceau Hi, this operator has changed in a way that breaks retro. Is this expected?